src/libcore/num/mod.rs

```diff
@@ -2328,7 +2328,7 @@ macro_rules! uint_impl {
         }
 
         /// Returns the smallest power of two greater than or equal to `self`.
-        /// Unspecified behavior on overflow.
+        /// More details about overflow behavior can be found in [RFC 560].
```

Comment:
I agree that linking to RFC might not be a good idea.

From what I understand about overflow behavior (correct me if I'm wrong)
1. Compiler can detect literal overflow at compile time, this can be disabled using 
`#[allow(overflowing_literals)]` [https://is.gd/M6Oxnf](https://is.gd/M6Oxnf)
2. In debug mode, compiler by default places some checks which panics on overflow. While in release mode it wraps on overflow. [https://is.gd/gMmhFo](https://is.gd/gMmhFo)
3. Overflow checking can be turned off at compile time using `-Z force-overflow-checks=off` flag.

I could not find any page in doc which documents behavior on overflow so I linked to RFC. Let me know if there is a page which should be linked.

Or we could just remove this section because overflow behavior across the whole program should now be consistent